docs/_snippets/webhook-signature-validation.mdx

```diff
@@ -0,0 +1,18 @@
+```typescript
+
+const signature = req.headers["x-metriport-signature"];
+  const metriportApi = new MetriportMedicalApi(apiKey, {
+    baseAddress: apiUrl,
+  });
+  try {
+    if (metriportApi.verifyWebhookSignature(whKey, req.body, signature)) {
```

Comment:
Now the way stripe does it.

Reply:
🤩 

nit: `const signature = req...` is missing identation